hack/logextractor.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# This script can be used to extract the failure cluster logs tarball from the PVC created for an install pod.
+#
+# Usage: ./logextractor.sh [LOGS_PVC_NAME] [DEST_DIR]
+#
+
+echo "apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    run: logextractor
+  name: logextractor
+spec:
+  selector:
+    matchLabels:
+      run: logextractor
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        run: logextractor
+    spec:
+      volumes:
+      - name: logsvol
+        persistentVolumeClaim:
+          claimName: $1
+      containers:
+      - image: centos/httpd-24-centos7
+        imagePullPolicy: Always
+        name: logextractor
+        resources: {}
+        volumeMounts:
+        - mountPath: "/logs"
+          name: logsvol
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File" | kubectl apply -f -
+
+
+POD_NAME=$(kubectl get pod -l run=logextractor -o jsonpath="{.items[0].metadata.name}")
+echo "waiting for pod to be ready: $POD_NAME"
+
+kubectl wait --for=condition=Ready --timeout=120s pod/$POD_NAME
+
+echo "copying logs to ./logs"
+kubectl cp $POD_NAME:/logs $2
+
+echo "deleting logextractor deployment"
+kubectl delete deployment logextractor
```

Comment:
IMO we should use `oc` binary to keep things consistent. We suggest `oc` instead of `kubectl` throughout our documentation.